pkg/operator/resources/validations.go

```diff
@@ -93,10 +93,10 @@ func ValidateClusterAPIs(apis []userconfig.API, projectFiles spec.ProjectFiles)
 		api := &apis[i]
 		if api.Kind == userconfig.SyncAPIKind {
 			if err := spec.ValidateAPI(api, projectFiles, types.AWSProviderType, config.AWS); err != nil {
-				return errors.Wrap(err, api.Identify())
```

Comment:
I think the wrapping of `api.Identify()` should remain at this level, and should be removed from all levels below (e.g. `ValidateAPISplitter()`, and presumably somewhere else which prompted you to make this change).

In general, we've found the most maintainable pattern is to wrap error context info at the highest level possible.

Reply:
As per our conversation, the requested changes have been addressed.